src/ethereum/ethNetwork.ts

```diff
@@ -1760,9 +1772,9 @@ export class EthereumNetwork {
     if (evmScanApiServers.length > 0) {
       blockheight.push(this.checkBlockHeightEthscan)
       nonce.push(this.checkNonceEthscan)
-      txs.push(this.checkTxsEthscan)
       tokenBal.push(this.checkTokenBalEthscan)
     }
+    txs.push(this.checkTxsEthscan) // We'll fake it if we don't have a server
```

Comment:
I would have left the line where it was, but then included an else clause with the line copied.